extensions/amp-link-rewriter/0.1/test/test-noscope-document.js

```diff
@@ -0,0 +1,171 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import helpersMaker from './test-helpers';
+
+describes.fakeWin(
+  'amp-link-rewriter',
+  {
+    win: {
+      location: 'http://partnersite.com/123',
+    },
+    amp: {
+      extensions: ['amp-link-rewriter'],
+    },
+  },
+  (env) => {
+    let helpers;
+
+    beforeEach(() => {
+      helpers = helpersMaker(env);
```

Comment:
Why are you passing env to helpersMaker?

Reply:
@micajuine-ho to be honest, I have the same question :sweat_smile: 

I've just copied this part from test-amp-link-rewriter.js this argument already was added to this function.

Reply:
Yeah, I don't think, it's necessary (could be wrong). Otherwise I like the improvements to the tests.

Reply:
I've removed it from both tests.